src/main/scala/devices/debug/SBA.scala

```diff
@@ -355,7 +355,6 @@ class SBToTL(implicit p: Parameters) extends LazyModule {
     tl.c.valid := false.B
     tl.e.valid := false.B
 
-    assert (!tl.d.valid || tl.d.ready, "Debug module not ready to accept TL response") // assert tl.d.valid |-> tl.d.ready
```

Comment:
if we are going to delete this assertion can we please add a cover property instead to show that we do handle it correctly.